src/components/MissionsList.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Missions from './Missions';
+import { fetchMissions } from '../redux/missions/missionsSlice';
+
+function MissionsList() {
+  const { missions } = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
```

Comment:
Great job on using useDispatch hook. To avoid re-renders I recommend you to wrap the dispatch in a if check that looks for the isLoading and an empty store so the dispatch gets called. ie.

```
useEffect(() => {
    if (isLoading && !rocketsData.length) {
      dispatch(fetchRocketsAPI());
    }
  }, [dispatch, isLoading, rocketsData.length]);
```